tinyagentos/projects/project_store.py

```
@@ -63,14 +64,26 @@ class ProjectStore(BaseStore):
-            # Column already exists on fresh installs (created by SCHEMA).
-            pass
+        for col_def in (
+            "ALTER TABLE project_members ADD COLUMN can_edit_canvas INTEGER NOT NULL DEFAULT 0",
+            "ALTER TABLE project_members ADD COLUMN is_lead INTEGER NOT NULL DEFAULT 0",
+        ):
+            try:
+                await self._db.execute(col_def)
+                await self._db.commit()
+            except Exception:
+                # Column already exists on fresh installs (created by SCHEMA).
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)